discovery-provider/src/queries/get_remixable_tracks.py

```diff
@@ -15,6 +15,7 @@ def get_remixable_tracks(args):
         # Subquery to get current tracks that have stems
         remixable_tracks_subquery = (
             session.query(Track)
+                .select_from(Track)
```

Comment:
i don't think this would be necessary, but it shouldn't impact anything else i suppose

Reply:
I don't think so either, was trying things before.